src/main/java/seedu/chatteroo/parser/Parser.java

```diff
@@ -0,0 +1,90 @@
+package seedu.chatteroo.parser;
+
+import seedu.chatteroo.commands.AddCommand;
+import seedu.chatteroo.commands.Command;
+import seedu.chatteroo.commands.DeleteCommand;
+import seedu.chatteroo.commands.ExitCommand;
+import seedu.chatteroo.commands.FindCommand;
+import seedu.chatteroo.commands.MarkCommand;
+import seedu.chatteroo.commands.PrintCommand;
+import seedu.chatteroo.commands.UnmarkCommand;
+
+
+import seedu.chatteroo.tasks.Task;
+import seedu.chatteroo.tasks.ToDo;
+import seedu.chatteroo.tasks.Deadline;
+import seedu.chatteroo.tasks.Event;
+
+/**
+ * Parses the user input and returns the corresponding command.
+ */
+public class Parser {
+    /**
+     * Constructor for the Parser class.
+     */
+    public Parser() {
+    }
+
+    /**
+     * Parses the user input and returns the corresponding command using a switch statement.
+     * @param input The user input.
+     * @return The corresponding command.
+     * @throws Exception If the input is invalid.
+     */
+    public static Command parseInput(String input) throws Exception {
+        String[] inputArr = input.split(" ");
+        String command = inputArr[0].toUpperCase();
```

Comment:
I like how you used switch statements with correct indentation here. You may consider adding a new line after String command for more clarity